XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizardPrecheckPage.cs

```diff
@@ -216,7 +220,8 @@ public override string HelpID
                     checkGroup = checks[checks.Count - 1].Value;
                     foreach (Host host in SelectedServers)
                     {
-                        checkGroup.Add(new UpgradingFromTampaAndOlderCheck(host));
+                        if(!Helpers.ClearwaterOrGreater(host))

```

Comment:
Missing the licence check here; the version check applies also in lines 231 and 239